lib/str-utils.h

```diff
@@ -167,9 +167,9 @@ ch_isalpha(gchar c)
 }
 
 static inline gboolean
-ch_isascii(gchar c)
+ch_isascii(guchar c)
 {
-  return c >= 0 && c <= 127;
+  return c <= 127;
```

Comment:
I expected the compiler to omit the superfluous condition, but I found it more reassuring to have the condition
anyway.

Changing the signature of the function I do not like that much, as C uses signed chars by default and everything is signed in  string handling.

I'd prefer an appropriate cast with this in mind.